tests/common/platform/device_utils.py

```diff
@@ -353,6 +353,23 @@ def verify_no_coredumps(duthost, pre_existing_cores):
                                                                                   coredumps_count))
 
 
+@handle_test_error
+def verify_yang(duthost):
+    """
+    Verify yang over running config
+    """
+    logging.info("Verify yang over running config")
+
+    # Strict yang validation is supported from 2025
+    strict_yang_validation = True
+    non_strict_versions = ["2019", "2020", "2021", "2022", "2023", "2024"]
```

Comment:
>2024 [](http://example.com/codeflow?start=68&length=4)

If we use 202405, then we can match the whole string instead of substring?

Reply:
Do you mean to put 202405 to strict yang?